### DISCUSSION TITLE
Morph plug-in bug

### DISCUSSION THREAD

**Discussion by codepilotsf** [alpinejs/alpine#3418] [7. Bugs (archived)]
Sorry if this is a duplicate. I can't see the bug I just reported 10 mins ago.

Very unexpected behavior with Morph Plugin using basic example shown on the plugin page. 

Simply change the value of `message` in the toHTML and then remove the `h2` tag from the example. Bizarrely, the presence of that H2 tag radically changes behavior of the state.

**Comment by SimoTod** [comment]
FYI: https://github.com/alpinejs/alpine/discussions/1423
